Packs/CommonScripts/Scripts/SetIfEmpty/SetIfEmpty.py

```diff
@@ -10,7 +10,13 @@ def get_value_to_set(args):
     if value is None or (apply_if_empty and is_value_empty(value)):
         value = args.get('defaultValue')
 
-    return encode_string_results(value)
+    if isinstance(value, list):
+        for i, item in enumerate(value):
+            value[i] = encode_string_results(item)
```

Comment:
```suggestion
        for key, item in enumerate(value):
            value[key] = encode_string_results(item)
```

Reply:
since `value` is a list, `i` stands for index.